cdap-proto/src/main/java/co/cask/cdap/proto/id/EntityId.java

```diff
@@ -185,7 +185,8 @@ public MetadataEntity toMetadataEntity() {
       // application so append it
       extractedParts.add(new MetadataEntity.KeyValue(MetadataEntity.VERSION, version));
     }
-    if (entityType == EntityType.PROGRAM || entityType == EntityType.SCHEDULE) {
+    if (entityType == EntityType.PROGRAM || entityType == EntityType.SCHEDULE ||
+      entityType == EntityType.PROGRAM_RUN || entityType == EntityType.FLOWLET) {
```

Comment:
The linebreak should be before the `||`, not after.

```java
if (condition1 
  || condition2) {
}
```

Reply:
Done.